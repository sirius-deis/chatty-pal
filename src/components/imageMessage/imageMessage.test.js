import { render, screen } from "@testing-library/react";
import ImageMessage from "./imageMessage";

const message = {
  src: "image.jpg",
  createAt: new Date("2024-07-17T21:08:30").toISOString()
}

describe("ImageMessage component", () => {
  it("should match snapshot", () => {
    const { container } = render(<ImageMessage message={message} />);
    expect(container).toMatchSnapshot();
  });

  it("renders an image", () => {
    render(<ImageMessage message={message} />);
    const img = screen.getByAltText(/image/i)
    expect(img).toBeInTheDocument();
    expect(img).toHaveAttribute("src", message.src);
  });
})