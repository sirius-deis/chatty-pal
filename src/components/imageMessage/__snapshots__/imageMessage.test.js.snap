// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageMessage component should match snapshot 1`] = `
<div>
  <div
    class="sc-aXZVg bJeYAM"
  >
    <img
      alt="image.jpg"
      src="image.jpg"
    />
  </div>
</div>
`;
